boards/xtensa/m5stack_atoms3/board.cmake

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(NOT "${OPENOCD}" MATCHES "^${ESPRESSIF_TOOLCHAIN_PATH}/.*")
+  set(OPENOCD OPENOCD-NOTFOUND)
+endif()
+find_program(OPENOCD openocd PATHS ${ESPRESSIF_TOOLCHAIN_PATH}/openocd-esp32/bin NO_DEFAULT_PATH)
```

Comment:
Since debugging is not supported, maybe consider reducing this file to single line:
`include(${ZEPHYR_BASE}/boards/common/esp32.board.cmake)`